libs/blocks/caas/utils.js

```diff
@@ -199,12 +199,22 @@ export const decodeCompressedString = async (txt) => {
 };
 
 export const loadCaasFiles = async () => {
-  const version = new URL(document.location.href)?.searchParams?.get('caasver') || 'stable';
+  const searchParams = new URLSearchParams(document.location.search);
+  const version = searchParams?.get('caasver') || 'stable';
+  let cssFile = `https://www.adobe.com/special/chimera/caas-libs/${version}/app.css`;
+  let jsFile = `https://www.adobe.com/special/chimera/caas-libs/${version}/main.min.js`;
+
+  // for caas local development
+  const host = searchParams?.get('caas-host');
```

Comment:
Nit: one query param is `caasver` (no hyphen), the other one is `caas-host` (with hyphen). Could we use the same naming rule for both?

Reply:
Good point. Updated